src/index.html

```diff
@@ -9,6 +9,55 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          class="logo-img"
+          src="./images/logo.png"
+          alt="asd"
```

Comment:
The `alt` attribute in the `img` tag should have a meaningful description of the image for better accessibility.